src/components/Animal/Animal.js

```diff
@@ -1,3 +1,12 @@
-export default function Animal() {
-  return <></>;
+import React from 'react';
+import './Animal.css';
+
+export default function Animal({ name, says, top, left, type }) {
+  return (
+    <div className="animal" style={{ top, left }}>
+      <img alt={name} src={`${process.env.PUBLIC_URL}/animals/${type}.svg`} />
```

Comment:
You don't need the process.env.PUBLIC_URL.

animals/${props.type}.svg should work